library/core/src/ptr/const_ptr.rs

```diff
@@ -1347,16 +1366,26 @@ impl<T: ?Sized> *const T {
     /// # Panics
     ///
     /// The function panics if `align` is not a power-of-two (this includes 0).
+    // #[cfg(not(bootstrap))] -- Calling this function in a const context from the bootstrap
+    // compiler will always return false.
```

Comment:
This comment is not necessary.

Reply:
However, it *is* necessary to explain that in `const`, this function is *approximate* and can return `false` even for pointers that, when checked at runtime, turn out to be aligned.

Reply:
I've added this to the docs.